<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>МНК</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>10</y>
      <width>231</width>
      <height>34</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="nLabel">
       <property name="text">
        <string>Кол-во точек: </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="xLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="yLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="mnk_button">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Найти значения</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>71</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="xLayout"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>71</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="yLayout"/>
   </widget>
   <widget class="QLabel" name="sLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>S</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>83</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="sLayout"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>360</y>
      <width>171</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="dbLayout">
     <item>
      <widget class="QPushButton" name="outbaseButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Получить из таблицы</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="inbaseButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Занести в таблицу</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="zLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>16</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>61</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="zLayout"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>50</y>
      <width>111</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="resLayout"/>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>59</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Полином</string>
    </property>
   </widget>
   <widget class="QComboBox" name="polynomeBox">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>x + y</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>x + y^2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>x^2 + y</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>x^2 + y^2</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
